libs/blocks/ost/ost.css

```diff
@@ -0,0 +1 @@
+/* created due of block nature in Milo */
```

Comment:
FWIW, I started #752 to discuss this requirement.

Reply:
Since the conversation is still ongoing may I suggest we leave this for now for the sake of removing the 404?

Reply:
File is no longer empty :)